contracts/SpokePool.sol

```diff
@@ -540,7 +541,7 @@ abstract contract SpokePool is
         address exclusiveRelayer,
         uint32 quoteTimestamp,
         uint32 fillDeadline,
-        uint32 exclusivityDeadline,
+        uint32 exclusivityDeadlineOffset,
```

Comment:
From the perspective of a depositor I think `exclusivityWindow` or `exclusivityPeriod` might be more self-explanatory than `exclusivityDeadlineOffset`.

Reply:
I think period is the best, yeah